### DISCUSSION TITLE
Solved

### DISCUSSION THREAD

**Discussion by mariocavaiola** [google/model-viewer#4194] [General]
Works till few hours ago, now It stopped without modify my code. What's that?
EDIT: works again

**Comment by WadeMorrison** [comment]
Same, keep getting CORS error now. But was working fine a few hours ago.

**Reply by WadeMorrison** [reply]
Working now. Solved

**Comment by elalish** [comment]
Were you using `unpkg` to pull the model-viewer library? That CDN seems to be pretty unreliable these days. See https://modelviewer.dev/docs/faq.html#entrydocs-technical-questions-cdn

**Reply by WadeMorrison** [reply]
Yeah I was using that, and then replaced it with the [https://ajax.googleapis.com/ajax/libs/model-viewer/3.0.1/model-viewer.min.js](https://ajax.googleapis.com/ajax/libs/model-viewer/3.0.1/model-viewer.min.js), back to normal now.
